gpio/zynq7000-xgpio/README.md

```diff
@@ -0,0 +1,19 @@
+# zynq7000-xgpio
```

Comment:
I think the better place for documentation is the `phoenix-rtos-doc` repo.

Reply:
Where? We don't have specific drivers documentation in phoenix-rtos-doc repository: https://github.com/phoenix-rtos/phoenix-rtos-doc/tree/master/devices.

Also, this isn't really a documentation, it's just a brief usage info - it's a README ;)

Reply:
Place will be made and READMEs will be moved in the near future